### DISCUSSION TITLE
ant_media_flutter: ^1.3.1 upgrade than video player showing black screen.

### DISCUSSION THREAD

**Discussion by arupbkp** [ant-media/Ant-Media-Server#6273] [General]
How to solve video player black screen problem.
![Simulator Screenshot - iPhone 15 Pro Max - 2024-04-16 at 11 16 40](https://github.com/ant-media/Ant-Media-Server/assets/116704492/c0209237-fb39-41a1-b211-8bd6a5f51f41)

**Comment by yashtandon113** [comment]
@arupbkp 
Thanks for reporting the issue.

We will investigate and share the update with you.

**Reply by arupbkp** [reply]
Finally fix my issue .. ant_media_flutter: ^1.4.0
